code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -655,6 +655,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 			to_chat(M, "[FOLLOW_LINK(M, user)] [ghost_message]")
 	// Log in the talk log
 	log_talk(user, "[key_name(user)] (PDA: [initial(name)]) sent \"[message]\" to [target_text]", LOGPDA)
+	to_chat(user, "Message sent to [target_text]: \"[message]\"")

```

Comment:
to_chat(user, "<span class='info'>Message sent to [target_text]: \"[message]\"</span>")